app/src/main/java/com/example/nikit/eventsapp/LoginActivity.java

```diff
@@ -4,36 +4,36 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.AppCompatButton;
 import android.util.Log;
-import android.view.View;
-import android.widget.Button;
 import android.widget.EditText;
 import android.widget.Toast;
 
 import com.example.nikit.eventsapp.model.Login;
-import com.example.nikit.eventsapp.model.LoginResponse;
 import com.example.nikit.eventsapp.rest.ApiClient;
-import com.example.nikit.eventsapp.rest.ApiInterface;
 import com.example.nikit.eventsapp.utils.ConstantStrings;
 import com.example.nikit.eventsapp.utils.SharedPreferencesUtil;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
-import retrofit2.Call;
-import retrofit2.Callback;
-import retrofit2.Response;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.schedulers.Schedulers;
 
 public class LoginActivity extends AppCompatActivity {
-    ApiInterface apiService;
     @BindView(R.id.username_et)
     protected EditText usernameET;
     @BindView(R.id.password_et)
     protected EditText passwordET;
     @BindView(R.id.login_btn)
-    protected Button loginBtn;
+    protected AppCompatButton loginBtn;
+
```

Comment:
Why appcompat?

Reply:
@nikit19 In the xml file we do use an appcompat button. Won't matter though if I would change this to a Button. Just to be specific